src/components/mainPage/MiddleBlock.vue

```diff
@@ -35,13 +36,24 @@ export default {
   data() {
     return {
       moviesFoundMessage: I18Y[LOCALE].MOVIES_FOUND,
-      sortByMessage: I18Y[LOCALE].SORT_BY,
-      firstButtonMessage: I18Y[LOCALE].RELEASE_DATE,
-      secondButtonMessage: I18Y[LOCALE].RATING,
+      sortByMessage: I18Y[LOCALE].SORT_BY.toUpperCase(),
+      firstButtonMessage: I18Y[LOCALE].RELEASE_DATE.toUpperCase(),
+      secondButtonMessage: I18Y[LOCALE].RATING.toUpperCase(),
       firstOption: RELEASE_DATE_OPTION,
       secondOption: RATING_OPTION,
       options: SORT_INPUT
     };
+  },
+  computed: {
+    ...mapGetters(["moviesCount"]),
+    foundMoviesMessage() {
+      if (!this.moviesCount) {
+        return "No movies found";
+      } else if (this.moviesCount === 1) {
+        return this.moviesCount + " " + I18Y[LOCALE].MOVIE_FOUND;
```

Comment:
Consider defaulting to template literals.

Reply:
Fixed